lib/routes/lightning_address/lightning_address_page.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:async';
+
+import 'package:c_breez/bloc/input/input_bloc.dart';
+import 'package:c_breez/bloc/webhooks/webhooks_bloc.dart';
+import 'package:c_breez/routes/create_invoice/widgets/loading_or_error.dart';
+import 'package:c_breez/routes/create_invoice/widgets/successful_payment.dart';
+import 'package:c_breez/widgets/address_widget.dart';
+import 'package:c_breez/widgets/transparent_page_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:logging/logging.dart';
+import 'package:c_breez/widgets/back_button.dart' as back_button;
+
+final _log = Logger("LnurlpayPageState");
+
+class LightningAddressPage extends StatefulWidget {
+  const LightningAddressPage();
+
+  @override
+  State<StatefulWidget> createState() {
+    return LightningAddressPageState();
+  }
+}
+
+class LightningAddressPageState extends State<LightningAddressPage> {
+  @override
+  void initState() {
+    super.initState();
+    final webhookBloc = context.read<WebhooksBloc>();
+    webhookBloc.refreshLnurlPay();
+    context.read<InputBloc>().trackPayment(null).then((value) {
```

Comment:
What if we receive another payment? How does this work in terms of reliability?

Reply:
We don't really have a way to associate a specific payment. Worst case (which is very rare) we will exit this screen on payment and show the payment receive screen (which sounds OK to me).

Reply:
Perhaps we can pass a flag here, `isLnurlPay` and check if the next received payment is a ln payment on trackPayment as we have payment details there. Either way, it's OK for me.

Reply:
The trackPayment always notifies only for Lnpayment so we don't need the flag.